neqo-transport/src/fc.rs

```diff
@@ -171,8 +171,8 @@ where
 {
     /// The thing that we're counting for.
     subject: T,
-    /// Max amount of outstanding data
-    max_wnd: u64,
+    /// The maximum amount of items that can be active (e.g., the size of the receive buffer).
+    max_acive: u64,
```

Comment:
typo: `max_ac[t]ive`
```suggestion
    max_active: u64,
```

Reply:
it was late evening  :(